README.md

```diff
@@ -1,3 +1,8 @@
+# Work in Progress: adding support for Mistral
```

Comment:
We should remove this text now that we're moving it up to an almost main branch xD

Reply:
To clarify, yes this PR will be updated in order to merge it to the main branch along with Llama 2 (GQA) support, docs + clean configs and all!